dockerfiles/ubuntu/obam-uk/Dockerfile

```diff
@@ -0,0 +1,104 @@
+ARG BASE_DOCKER_IMAGE_REGISTRY="docker.wso2.com"
+ARG BASE_PRODUCT_VERSION=3.0.0.0
+
+# set base Docker image to WSO2 Open Banking API Manager Ubuntu Docker image
+FROM ${BASE_DOCKER_IMAGE_REGISTRY}/wso2-obam:${BASE_PRODUCT_VERSION}
+
+# set Docker image build arguments
+# build arguments for user/group configurations
+ARG USER=wso2carbon
+ARG USER_ID=802
+ARG USER_GROUP=wso2
+ARG USER_GROUP_ID=802
+ARG USER_HOME=/home/${USER}
+# build arguments for WSO2 product installation
+ARG WSO2_SERVER_NAME=wso2am
+ARG WSO2_SERVER_VERSION=4.0.0
+ARG WSO2_OB_TOOLKIT_NAME=wso2-obam-toolkit-uk
+ARG WSO2_OB_TOOLKIT_VERSION=1.0.0
+ARG WSO2_SERVER=${WSO2_SERVER_NAME}-${WSO2_SERVER_VERSION}
+ARG WSO2_OB_TOOLKIT=${WSO2_OB_TOOLKIT_NAME}-${WSO2_OB_TOOLKIT_VERSION}
+ARG WSO2_SERVER_HOME=${USER_HOME}/${WSO2_SERVER}
+ARG WSO2_OB_TOOLKIT_DIR=${WSO2_SERVER_HOME}/${WSO2_OB_TOOLKIT}
+ARG WSO2_OB_TOOLKIT_DIST_URL
+# build argument for MOTD
+ARG MOTD="\n\
+Welcome to WSO2 Docker resources.\n\
+------------------------------------ \n\
+This Docker container comprises of The WSO2 Open Banking Solution, with its latest GA release \n\
+which is under The WSO2 Commercial License 1.0. \n\
+Read more about The WSO2 Commercial License, Version 1.0 here @ http://wso2.com/licenses \n"
+
+# configuration properties for deployment.toml file
+# server hostnames
+ARG IS_HOSTNAME=obiam
+ARG APIM_HOSTNAME=obam
+ARG BI_HOSTNAME=localhost
+# databases related properties
+ARG DB_TYPE=mysql
+ARG DB_USER=root
+ARG DB_PASS=root
+ARG DB_HOST=mysql
+ARG DB_DRIVER=com.mysql.jdbc.Driver
+ARG DB_APIMGT=openbank_apimgtdb
+ARG DB_USER_STORE=openbank_userdb
+ARG DB_AM_CONFIG=openbank_am_configdb
+ARG DB_GOV=openbank_govdb
+ARG DB_IS_CONFIG=openbank_iskm_configdb
+ARG DB_OPEN_BANKING_STORE=openbank_openbankingdb
+ARG DB_OB_REPORTING=openbank_ob_reporting_statsdb
+ARG DB_OB_REPORTING_SUMMARIZED=openbank_ob_reporting_summarizeddb
+
+# change user to root for elevated permissions to override inherited non-root user
+USER root
+
+# add Open Banking artifacts to base product
+RUN \
+    wget -O ${WSO2_OB_TOOLKIT}.zip "${WSO2_OB_TOOLKIT_DIST_URL}" \
+    && unzip -d ${WSO2_SERVER_HOME}/ ${WSO2_OB_TOOLKIT}.zip \
+    && rm -f ${WSO2_OB_TOOLKIT}.zip
+
+# set configurations
+# read deployment.toml file
+ARG DEPLOYMENT_TOML_FILE=${WSO2_OB_TOOLKIT_DIR}/repository/resources/deployment.toml
+RUN \
+    cp ${WSO2_OB_TOOLKIT_DIR}/repository/resources/${WSO2_SERVER}-deployment-uk.toml ${DEPLOYMENT_TOML_FILE} \
```

Comment:
Lets try to come up with a way to avoid referencing "uk" here. Toolkit dockefile should be made generic so that the same  dockerfile can be used for any toolkit

Reply:
Since uk is not referenced everywhere, it is fine to keep this